### ISSUE TITLE
Transmission of the Lizard Rodeo

### ISSUE THREAD

**Issue by Dudejo** [Mogli12/GearboxAddon#383]
Hi again.

Have you considered converting the Lizard Rodeo into a manual transmission truck? When in 1st person view, the truck has 3 center pedals and the gear lever has a round rubber base typical of manual shifters.

Trucks of the Rodeo's apparent generation with automatic transmissions would have a steering column-mounted selector on the right-hand side. However, the Rodeo only has the left-side selector for operating flashers, wipers, etc.

**Comment by Mogli12** [comment]
Hi,

You can switch from automatic mode to manual mode. But the torque converter will stay.

Regards,
Stefan

**Comment by Dudejo** [comment]
I know I can do that.

It's just that the truck ISN'T an automatic...

**Comment by Mogli12** [comment]
Hi,

Ok, I thought that the lizard rodeo is somehow similar to an Ford F-series 5th gen. So I tried to emulate an 3-speed automatic (C4) and a 4 speed manual (T18) transmission. The ratios between the gears should be realistic. But I did not have good information about the final ratio.

Regards, Stefan

**Comment by Dudejo** [comment]
I found this Ford sales brochure which provides the appropriate axle ratios for a 5th generation Ford truck.

[http://paintref.com/graphics/brochure/1970fordtruck_pickup_12.jpg](url)

Assuming this is a light duty 4x4 truck with LSD, the axle ratios would be 3.50 or 4.09. The tire size for the Lizard Rodeo, if it matters, is 280/55R16.

Also, according to the following link, the gear ratios for the T18 are off.

[http://www.novak-adapt.com/knowledge/transmissions/manual/t18-t19](url)

In your mod, the First gear has a rated speed of 35 km/h. This is only possible with the Chrysler Jeep model of the heavier duty Ford T19. The Ford T18 uses a much lower gear that would result in a rated speed of 22 km/h.

**Comment by Mogli12** [comment]
I adjusted the gear ratios of the manual transmission.

**Comment by Dudejo** [comment]
I noticed the automatic has a shorter gear ratio. Is this to help the torque converter stay locked?

**Comment by Mogli12** [comment]
Good idea. Maybe we can have different axle speed ratios with and w/o splitter. Let' see.

**Comment by Dudejo** [comment]
wait, nevermind about the gear splitter.

having failed to read the fine print, I just noticed that the device cannot work while 4WD is active. There is simply no space to install the unit between the transmission and transfer case so they only install it on the rear axle side.

This may also explain why no 4WD truck from that era use overdrive transmissions...

**Comment by Dudejo** [comment]
It would roughly look like this with an Atlas II.

While building this, I also realized that the inverseRatio flag does not work when used in a secondary Range flag. The inverseRatio flag only works with the main gears or the primary range.

			<transmission name="C4/Atlas II" torqueConverterProfile="oldCar" speedLimiter="false" maxRpmThrottle="0.5" fuelPerDistanceMinSpeed="10" minTargetRpm="1000" torqueConverter="true" clutchCloseRpm="1700" torqueConverterMaxFactor="2.14">
				<blowOffVentilSound volume="0"/>
			
				
				<gears automatic="true" axleSpeed="4.11" tireWidth="280" tireRatio="55" rimDiameter="16" defaultGear="1" shiftTimeMs="0" shiftEffect="false" clutchRatio="0" manualClutch="false" startInSmallest="true">
					<gear name="1"  forwardOnly="true" inverseRatio="2.46"/>
					<gear name="R" reverseOnly="true" inverseRatio="2.18"/>
					<gear name="2"  forwardOnly="true" inverseRatio="1.46"/>
					<gear name="3"  forwardOnly="true" inverseRatio="1.00"/>
				</gears>

			<ranges defaultRange="2" reverseReset="false" shiftTimeMs="3000" shiftNoThrottle="true">
				<range inverseRatio="3.80" name="Lo" />
				<range inverseRatio="1.00" name="Hi" />
			</ranges>

			<ranges defaultRange="2" reverseReset="false" onlyStopped="true">
			
				<range ratio="0.36765" name="L" />
				<range ratio="1.00" name="H" />
			</ranges>
			</transmission>

**Comment by Mogli12** [comment]
Hi,

I think I want to keep the engines as they are to have them as close as possible to the vanilla configurations. 

Regards,
Stefan

**Comment by Dudejo** [comment]
do you mean the engine torque templates that I sent you by e-mail?

I tuned them to match the in-game power ratings (NOT the engine names) so they should match pretty well.

the only potential concern is the low RPM limit of the 300 I6 I made. It needs a tall differential ratio to reach the rated top speed of 120 km/h

**Comment by Mogli12** [comment]
Ok, what do you think of v3.1 build 70?

**Comment by Dudejo** [comment]
assuming you wish to stay close to the stock 5th generation model, I would recommend simulating a Dana 24 transfer case, which has a 1.86:1 low gear ratio.

The 2.72 low gear ratio used in your mod will not appear until the late 1980s.
